.gitmodules

```diff
@@ -854,6 +854,9 @@
 [submodule "vendor/grammars/language-webassembly"]
 	path = vendor/grammars/language-webassembly
 	url = https://github.com/Alhadis/language-webassembly
+[submodule "vendor/grammars/language-ring"]
+	path = vendor/grammars/language-ring
+	url = https://github.com/MahmoudFayed/atom-language-ring
```

Comment:
Well, it actually *does* recognise `\h` sequences. ;) They just have an entirely unrelated meaning: horizontal whitespace characters. Spaces and tabs, but not newlines or carriage returns, et cetera.

Bane of my regex-obsessed existence.